backend/python/app/tests/services/test_story_service.py

```diff
@@ -362,49 +300,370 @@ def test_get_story_translation_tests(app, db, services):
     )
 
 
-def test_assign_user_as_reviewer():
-    pass
+def test_assign_user_as_reviewer(app, db, services):
+    translator_obj = create_translator(db)
+    reviewer_obj = create_reviewer(db)
+    story_obj = db_session_add_commit_obj(
+        db,
+        Story(
+            title="Title",
+            description="Description",
+            youtube_link="",
+            level=4,
+        ),
+    )
+    story_translation_obj = db_session_add_commit_obj(
+        db,
+        StoryTranslationAll(
+            story_id=story_obj.id,
+            language="ENGLISH_US",
+            stage="TRANSLATE",
+            translator_id=translator_obj.id,
+            reviewer_id=None,
+        ),
+    )
+    create_story_translation_contents(db, story_translation_obj)
+    story_translation = services["story"].get_story_translation(
+        story_translation_obj.id
+    )
 
+    assert story_translation_obj.reviewer_id == None
+    resp = services["story"].assign_user_as_reviewer(
+        user=reviewer_obj, story_translation=story_translation
+    )
+    assert resp["stage"] == "REVIEW"
```

Comment:
we actually gotta change this, so remove this line https://www.notion.so/uwblueprintexecs/assign_user_as_reviewer-should-not-change-story-translation-stage-5f769d3cd38b48ad83dfa3c159585308